includes/Meta/Comment.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Adds support for saving/retrieving values from comment meta.
+ *
+ * @package    SCF
+ * @since      6.5
```

Comment:
Can you adapt these annotations to say `SCF 6.5`, to comply with the changes made in #129 ?

Reply:
I though in that case, as the package was SCF already, we didn't need to specify just one line after. But happy to update that.

Reply:
Done!